admin/data/sql/migrations/106.do.add-familiarisation-pin.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE [mtc_admin].[pupil] ADD familiarisation_pin NVARCHAR(12) NULL
```

Comment:
we definitely don't want 2 pin fields.  the bit field distinguishes this.  We went through this in fridays discussion (you may have joined after it was discussed?)
also, we are moving to a new table `checkFormAllocation`.  I mentioned to the BAs that this card was supposed to just cover the UI, and not actually do the plumbing, as that will be done by an azure function, triggered by a queue message.

Reply:
Ah okay, I was under the impression that the code from e.g. this feature would be moved to the Azure function after it was working. Should I remove the "functionality" for pin generation and address it after the Azure function / queue message is implemented to do that?

Reply:
you can stub the familiarisation mode out for now.  Jon has already picked up the ticket that covers that part of it i believe.

The UI side of things looks good, so we can just wire it up when Jons done the other ticket 😎